nuxeo-drive-client/nxdrive/client/local_client.py

```diff
@@ -717,7 +717,13 @@ def move(self, ref, new_parent_ref, name=None):
             self.lock_ref(new_parent_ref, locker & 1 | new_locker)
 
     def is_inside(self, abspath):
-        return abspath.startswith(self.base_folder)
+        if abspath.startswith(self.base_folder):
+            return True
+        else:
+            # Computing path for temp file in windows
+            # during remote sync.
+            abspath = abspath.lstrip("\\?")
```

Comment:
I'm sure you tested it, but if the abspath is \\?\C:\Users\Bob\CPO\My Docs, should it strip "\\?\" prefix? 
Otherwise, \C\Users\Bob\CPO\My Docs does *not* start with C:\Users\Bob\CPO